src/irmin-git/atomic_write.ml

```diff
@@ -233,9 +232,10 @@ module Make (K : Key) (G : Git.S) = struct
     | _ -> false
 
   let test_and_set t r ~test ~set =
-    Log.debug (fun f ->
-        let pp = Fmt.option ~none:(Fmt.any "<none>") (Irmin.Type.pp Val.t) in
-        f "test_and_set %a: %a => %a" pp_branch r pp test pp set);
+    [%log.debug fun f ->
+      let pp = Fmt.option ~none:(Fmt.any "<none>") (Irmin.Type.pp Val.t) in
+      f "test_and_set %a: %a => %a" pp_branch r pp test pp set]
+    ;
```

Comment:
This is one of two cases where the new syntax is longer than the old one, and both are due to the same formatting weirdness for code of the form `[%ext fun ... -> ...]; ...`.

I've opened an OCamlformat issue about this [here](https://github.com/ocaml-ppx/ocamlformat/issues/1819).